ClosedXML/XLHelper_ASF.cs

```diff
@@ -0,0 +1,142 @@
+﻿// Keep this file CodeMaid organised and cleaned
+
+/* ====================================================================
+   Licensed to the Apache Software Foundation (ASF) under one or more
+   contributor license agreements.  See the NOTICE file distributed with
+   this work for additional information regarding copyright ownership.
+   The ASF licenses this file to You under the Apache License, Version 2.0
+   (the "License"); you may not use this file except in compliance with
+   the License.  You may obtain a copy of the License at
+       http://www.apache.org/licenses/LICENSE-2.0
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+==================================================================== */
+
+using System;
+
+namespace ClosedXML.Excel
+{
+    public static partial class XLHelper
+    {
+        /// <summary>
+        /// Creates a valid sheet name, which is conform to the rules.
+        /// Rules:
+        /// never null
+        /// minimum length is 1
+        /// maximum length is 31
+        /// doesn't contain special chars: : 0x0000, 0x0003, / \ ? * ] [
+        /// Sheet names must not begin or end with ' (apostrophe)
```

Comment:
In tooltips, line breaks are removed and the text without separators becomes hard to read. Please add commas or bullet points
![image](https://user-images.githubusercontent.com/19576939/86736609-b8441480-c044-11ea-983d-0fdb7f37af5e.png)